### PR TITLE
Workflow

### PR DESCRIPTION
**Pull request by apaasch**
Add customise tasks for:

1. Alpha extrapolation

- Prepare Asymmetry histograms (normalise inclusive histograms)
- Split into Standard Method and Forward Extrension
- Perform linear fit to widhts_A in dependency of alpha
- Store arrays into new pickle file

2. JER measurement

- Read in task structure from alpha extrapolation
- Get JER for Standard Method
- Get JET for Forward extension with respect to the SM
- Store in new pickle file

### CODE REVIEW

**Review by dsavoiu — CHANGES_REQUESTED** [review]
Thanks a lot for this extensive PR @apaasch! It's very good to see the JER workflow implemented as columnflow tasks. I went through some parts of this in more detail than others, but I think overall the proposed task structure is fine at this point. We discussed privately how it can be improved further, but we can leave that to a different PR.

For this one I have several suggestions, mostly regarding docstrings and variable naming, but also some related to code structure. Could you please go through these and check if they can be implemented?

File: dijet/tasks/alpha.py
Comment (apaasch) [reply dijet/tasks/alpha.py:149]: I would keep it for now. The edges go from 0-1, but we only want to go to 0.3 since here the linear dependency (normally) still holds. This topic will be addressed in a next PR when we move to a histogram based structure and define the limit (0.3) in the config.

File: dijet/tasks/alpha.py
Comment (apaasch) [reply dijet/tasks/alpha.py:81]: Will be changed

File: dijet/tasks/base.py
Comment (apaasch) [reply dijet/tasks/base.py:29]: Will be changed

File: dijet/tasks/jer.py
Comment (apaasch) [reply dijet/tasks/jer.py:42]: Will be changed

**Review by apaasch — COMMENTED** [review]
Thanks for the input. Most suggestion were added.

File: dijet/tasks/alpha.py
Comment (apaasch) [reply dijet/tasks/alpha.py]: In your suggestion you pass an integer as method. I fixed this and only use the dictionary in the run function but not in the get_norm_asymmetry

File: dijet/tasks/base.py
Comment (apaasch) [reply dijet/tasks/base.py]: In your suggestion l.184 you asked for h.axes["shift"]. h did not exist yet and I have changed it to the input histogram.

**Review by dsavoiu — CHANGES_REQUESTED** [review]
Thanks for the update! Looks OK to me, just a few loose ends left to tie up but once those are fixed I think we can merge.

File: dijet/tasks/alpha.py
Comment (dsavoiu) [reply dijet/tasks/alpha.py]: As discussed privately, let's keep this in mind for the next PR ;)

**Review by dsavoiu — APPROVED** [review]
Great, thanks for the speedy fixes! Looks like we can merge now 👍

File: dijet/tasks/alpha.py
Comment (dsavoiu) [reply dijet/tasks/alpha.py]: Discussed privately, `method` left as string in the end.

**Pull request merged**
